_docs/_latest/develop/dev-setup.md

```diff
@@ -173,32 +118,44 @@ You can find the `CPPFLAGS` and `LDFLAGS` information and export them as follows
     CPPFLAGS: -I/SOME/PATH/TO/openssl/include
   ```
 
-Then copy the `LDFLAGS` and `CPPFLAGS` values and export them, similar to the 
-following (use the values from your terminal, not "`/SOME/PATH/TO`"):
+Next, copy the `LDFLAGS` and `CPPFLAGS` values and export them, similar to the 
+following:
 
   ```bash
   $ export CPPFLAGS=-I/SOME/PATH/TO/openssl/include
   $ export LDFLAGS=-L/SOME/PATH/TO/openssl/lib
   ```
+In the above example, `/SOME/PATH/TO` represents the path specific to your
+system. Make sure to use the values from your terminal.
 
 ### Configuring Forseti settings
 
 Before you run Forseti, you need to edit the forseti_conf.yaml file, found in
```

Comment:
Does this mean I need to do everything that's on the Configuring Forseti page, or are there a sub-set of specific steps I should complete? (I'll suggest clarifying text depending.)